packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	comment as commentIcon,
+	commentAuthorAvatar as userIcon,
+	Icon,
+	check as resolvedIcon,
+} from '@wordpress/icons';
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { dateI18n, format, getSettings } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import PluginSidebar from '../plugin-sidebar';
+import { store as editorStore } from '../../store';
+
+const isBlockCommentExperimentEnabled =
+	window?.__experimentalEnableBlockComment;
+
+/**
+ * Renders the Collab sidebar.
+ */
+export default function CollabSidebar() {
+	const { threads } = useSelect( ( select ) => {
+		const meta = select( editorStore ).getEditedPostAttribute( 'meta' );
+		return {
+			threads: meta?.collab ? JSON.parse( meta.collab ) : false,
```

Comment:
This will return a new object on every select call, which will cause a re-render for every change to the editor store. JSON should be parsed on render instead, perhaps memoized.

Reply:
Also maybe we can use `useEntityProp` here too?